### ISSUE TITLE
Rework for Meteor 1.3?

### ISSUE THREAD

**Issue by demiters** [spacedrop/spacedrop#14]
Is this planned?

**Comment by timbrandin** [comment]
Yeah, I will add a lot of work on SpaceDrop the next month, have done a lot of work in a client project that I'll be refactoring in here.
